### PR TITLE
We want to have Python 3 compatibility.

### PR DESCRIPTION
**Pull request by sallner**
and testcoverage so i enabled tox as testrunner.

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ Changes
 1.6 (unreleased)
 ----------------
 
-- Nothing changed yet.
+- Drop support of Python 2.6 and claim support for Python 3.4, 3.5, 3.6 and PyPy.
```
Comment (icemac) [inline CHANGES.txt]: I think this should be version `2.0`.

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,28 @@
+[tox]
+envlist = coverage-clean, py27, py34, py35, py36, pypy, coverage-report
```
Comment (icemac) [inline tox.ini:2]: What about pypy3?

**Review by icemac — APPROVED** [review]
LGTM besides some questions.

File: tox.ini (hunk 2 above)
Comment (sallner) [reply tox.ini:2]: It does not work, because it still resembles Python 3.3.

File: CHANGES.txt (hunk 1 above)
Comment (sallner) [reply CHANGES.txt]: Has been Done

**Comment by icemac** [comment]
@janjaapdriessen I just released [zope.testing 4.6.0](https://pypi.org/project/zope.testing/4.6.0/) which includes  `IGNORE_EXCEPTION_MODULE_IN_PYTHON2.` Please add this version as minimum requirement and we'll see whether travis gets green.

**Pull request merged**